tests/scripts/run-throughput-perf.py

```diff
@@ -0,0 +1,361 @@
+#!/usr/bin/env python
```

Comment:
Coreclr has a python 2 dependency. None of the scripts in the repo to my knowledge are compatible with python 3.